serverless/virus-scanner/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const decodeS3URI = (uri: string) => {
```

Comment:
unused

Reply:
thanks! in [45cd743](https://github.com/opengovsg/FormSG/pull/6642/commits/45cd74377c49c5b7a318a3a67226643c39eb7c38)